app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -60,45 +60,40 @@ object EnvelopeReport {
 case class CreateEnvelopeRequest(callbackUrl: Option[String] = None,
                                  expiryDate: Option[DateTime] = None,
                                  metadata: Option[JsObject] = None,
-                                 constraints: Option[EnvelopeConstraintsUserO] = None)
+                                 constraints: Option[EnvelopeConstraintsUserO] = Some(EnvelopeConstraintsUserO()))
```

Comment:
```
EnvelopeConstraintsUserO(maxItems: Option[Int] = Some(100),
                                     maxSize: Option[String],
                                     maxSizePerItem: Option[String])
```
wrong type at `maxItems` should be reported with same type of error as wrong String format at `maxSize`